schema/rdl_schema_0.1.json

```diff
@@ -43,6 +47,15 @@
 								"Loss"
 							]
 						},
+						"disaster_identifier": {
+							"type": "string",
+							"description": "The GLIDE number associated with this event disaster. A globally unique identification code for the observed or inferred hazard event. GLIDE numbers are available at https://www.glidenumber.net/glide/public/search/search.jsp."
+						},
+						"disaster_date": {
+							"type": "string",
+							"description": "The date the disaster data as identified by it's GLIDE number was collected.",
```

Comment:
Is this the date that the disaster occurred, i.e. the disaster's date in GLIDE, or is it the date data that the data was collected?

Reply:
Continued in https://github.com/GFDRR/rdl-standard/issues/60